modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/IBlogPostRepository.cs

```diff
@@ -24,4 +26,6 @@ public interface IBlogPostRepository : IBasicRepository<BlogPost, Guid>
     Task<bool> SlugExistsAsync(Guid blogId, string slug, CancellationToken cancellationToken = default);
 
     Task<BlogPost> GetBySlugAsync(Guid blogId, string slug, CancellationToken cancellationToken = default);
-}
+
+    Task<List<CmsUser>> GetAuthorsHasBlogPosts(CancellationToken cancellationToken = default);
```

Comment:
Making this method name `GetAuthorsHasBlogPostsAsync` might be better

```suggestion
    Task<List<CmsUser>> GetAuthorsHasBlogPostsAsync(CancellationToken cancellationToken = default);
```